restcomm/restcomm.telephony/src/main/java/org/restcomm/connect/telephony/CallManager.java

```diff
@@ -533,35 +531,64 @@ private void invite(final Object message) throws IOException, NumberParseExcepti
 
                 if (isWebRTC(request)) {
```

Comment:
Also, if we execute extensions before `isWebRTC()` and proxy out, we can fail fast